deploy/hardware/generic.xml

```diff
@@ -11,10 +11,7 @@
     <apic/>
     <pae/>
   </features>
-  <cpu mode='custom' match='exact'>
-    <model fallback='allow'>kvm64</model>
-    <feature policy='require' name='vmx'/>
-  </cpu>
+  <cpu mode='host-passthrough'/>
```

Comment:
It is an option. But it requires all the roles to be duplicated, and so forth also the cloud config files.